Core/src/org/sleuthkit/autopsy/corecomponents/Bundle.properties

```diff
@@ -81,6 +81,11 @@ DataContentViewerHex.goToPageTextField.err=Invalid page number
 DataContentViewerHex.setDataView.errorText=(offset {0}-{1} could not be read)
 DataContentViewerHex.title=Hex
 DataContentViewerHex.toolTip=Displays the binary contents of a file as hexidecimal, with bytes that are displayable as ASCII characters on the right.
+DataContentViewerHTML.title=HTML
+DataContentViewerHTML.toolTip=Displays the rendered HTML from the file.
+DataContentViewerHTML.acceptedMIMEType=text/html
```

Comment:
I don't think this needs to be in the bundle.  It is never going to be internationalized is it?